src/query_builder.erl

```diff
@@ -38,7 +39,7 @@ build_lookup_query(Collection, Metric, Grouping) ->
     {ok, Query, Values}.
 
 build_lookup_query(Bucket, Metric, Where, Grouping)
-  when is_list(Metric) ->
+  when is_list(Metric); is_atom(Metric) ->
```

Comment:
Metric is `undefined` in the case that the `select all from bucket` query construction was used.  The immediate alternative I can think of is:
```
   when is_list(Metric); Metric =:= undefined
``` 

I did explore other approaches in terms of different tagged tuples {in... vs {all..., but this approach was cleanest.